pkg/rtc/participant.go

```diff
@@ -1739,6 +1762,9 @@ func (p *ParticipantImpl) removePublishedTrack(track types.MediaTrack) {
 	p.RemovePublishedTrack(track, false, true)
 	if p.ProtocolVersion().SupportsUnpublish() {
 		p.sendTrackUnpublished(track.ID())
+		if lmt, ok := track.(types.LocalMediaTrack); ok {
```

Comment:
should do this irrespective of client support for unpublish?

Reply:
It also happens on track onClose:

https://github.com/livekit/livekit/pull/3549/files#diff-21970cf376614865e67d05587cc879548c0b6e60e7e45e98b4cfd4c194e3daa6R2642-R2658

Reply:
I saw that. So, is this needed?

Reply:
Got it. But, I am trying to understand the need to do it here. Let the track close path take care of both cases.

Reply:
hmm but it is being removed from uptrackmanager's publishedTracks anyway

Reply:
Will keep the same behavior then

Reply:
Pushed the change, merge time?

Reply:
sounds good